src/NimBLERemoteCharacteristic.cpp

```diff
@@ -27,17 +27,18 @@
 
 # include <climits>
 
-typedef struct {
-    const NimBLEUUID* uuid;
-    void*             task_data;
-} desc_filter_t;
+struct desc_filter_t {
+    NimBLERemoteDescriptor* dsc;
+    const NimBLEUUID*       uuid;
+    void*                   taskData;
+};
```

Comment:
Since this is no longer a typedef can you rename it to `NimBLEDescriptorFilter` please?